### DISCUSSION TITLE
How to make changes

### DISCUSSION THREAD

**Discussion by pelotyo** [nxrighthere/UnrealCLR#119] [Help] [answered]
Hey I spend my all day trying to use this UnrealCLR.
I use default test scene that you team made and go to c# script and add some changes like changing the wave sound to other wave sound.
But don't get result about that, what I have to do to make changes...(´;o;｀)

**Comment by nxrighthere** [comment, marked as answer]
You have to compile managed assemblies with your changes and replace the previous version. Please, follow the instructions in the [manual](https://github.com/nxrighthere/UnrealCLR/blob/master/MANUAL.md#getting-started) to get started with the plugin.

**Reply by pelotyo** [reply]
how I can compile managed assemblies? What kind of version, you are talking ?What it's mean...

Sorry for my bad understanding. It's difficult to understand guide of programming in English I tried to find Japanese information about it but I can't find it. And I am trying to get success to share this information in Japanese for other user.
I see your demo movie that you can apply changes quickly but I try many things in many different way but it just go wrong...

**Reply by pelotyo** [reply]
I read the guide and do the steps but it's go wrong

**Reply by nxrighthere** [reply]
> how I can compile managed assemblies?

The simplest way to do that is to use .NET CLI, please, follow [this](https://docs.microsoft.com/en-us/dotnet/core/tools/) documentation to get familiar with the tool. 

> What kind of version, you are talking ?What it's mean...

If you are trying to modify the [tests](https://github.com/nxrighthere/UnrealCLR/tree/master/Source/Managed/Tests) project, then you have to replace assemblies in `%Project%/Managed/Tests` folder compiled by the installation tool with your modified version.

> Sorry for my bad understanding. It's difficult to understand guide of programming in English I tried to find Japanese information about it but I can't find it. And I am trying to get success to share this information in Japanese for other user.
I see your demo movie that you can apply changes quickly but I try many things in many different way but it just go wrong...

The workflow is very simple if you are familiar with .NET ecosystem. If not, then I recommend to start from [basics](https://www.youtube.com/watch?v=bEfBfBQq7EE).
